src/Sylius/Bundle/CoreBundle/Message/Admin/Account/ResetPassword.php

```diff
@@ -11,9 +11,8 @@
 
 declare(strict_types=1);
 
-namespace Sylius\Bundle\ApiBundle\Command\Admin;
+namespace Sylius\Bundle\CoreBundle\Message\Admin\Account;
 
-/** @experimental */
```

Comment:
should stay for a while

Reply:
I'm getting conflicting messages here 😢 <br>
@Zales0123 vs @lchrusciel 
You guys fight it out o/<br>
Personally I'd expect things in Core to strictly follow the BC rules.
I can find only a few other `experimental` tags in this scope, one is grid related and the other touches dashboard statistics. All of those had close to no changes in the past two years.

Reply:
I stand by my opinion :dancer: Resetting password was previously experimental because it was in ApiBundle, but as we provide it for the UI as well, I see no reason it should stay experimental... especially as we provide it in 1.12, which will not be released until the end of Q3 probably 🖖

Reply:
I agree with @Zales0123. IMO there's no good reason to have this class marked as experimental.